Documentation for SOS (#22117)
dotnet/docs: docs/core/diagnostics/index.md

### dotnet-sos

> [dotnet-sos](dotnet-sos.md) is used to install the [SOS debugging extension](sos-debugging-extension.md) on Linux or MacOS (or on Windows if using [Windbg/cdb](https://docs.microsoft.com/windows-hardware/drivers/debugger/debugger-download-tools)).

Reviewer: ```suggestion
[dotnet-sos](dotnet-sos.md) installs the [SOS debugging extension](sos-debugging-extension.md) on Linux and macOS (and on Windows if you're using [Windbg/cdb](/windows-hardware/drivers/debugger/debugger-download-tools)).
```

> [dotnet-sos](dotnet-sos.md) is used to install the [SOS debugging extension](sos-debugging-extension.md) on Linux or MacOS (or on Windows if using [Windbg/cdb](https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/debugger-download-tools)).

Reviewer 2: ```suggestion
[dotnet-sos](dotnet-sos.md) is used to install the [SOS debugging extension](sos-debugging-extension.md) on Linux or MacOS (or on Windows if using [Windbg/cdb](https://docs.microsoft.com/windows-hardware/drivers/debugger/debugger-download-tools)).
```
